docs/site/Server.md

```diff
@@ -256,6 +256,23 @@ app.basePath('/api');
 With the `basePath`, all REST APIs and static assets are served on URLs starting
 with the base path.
 
+### Configure the router
+
+The router can be configured to enforce `strict` mode as follows:
+
+1. `strict` is true:
+
+- request `/orders` matches route `/orders` but not `/orders/`
+- request `/orders/` matches route `/orders/` but not `/orders`
+
+2. `strict` is false (default)
+
+- request `/orders` matches route `/orders` first and falls back to `/orders`
```

Comment:
Should the last `/orders` be `/orders/`?

Reply:
Good catch.